Hearthstone Deck Tracker/Windows/NoteDialog.xaml.cs

```diff
@@ -61,5 +86,31 @@ private void CheckBoxEnterToSave_OnUnchecked(object sender, RoutedEventArgs e)
 			Config.Instance.EnterToSaveNote = false;
 			Config.Save();
 		}
+
+		private class NoteCard
+		{
+			public string Name { get; }
+			public string CountText { get; }
+			public Brush TextColor { get; }
+			public int Cost { get; }
+
+			public NoteCard() : this(null)
+			{
+			}
+
+			public NoteCard(TrackedCard tracked)
+			{
+				Card card = null;
+				if(tracked == null || !Cards.All.ContainsKey(tracked.Id))
+					card = Cards.All[NonCollectible.Neutral.Noooooooooooo];
+				else
+					card = Cards.All[tracked.Id];
+
+				Name = card.Name;
+				CountText = $"x{tracked.Count}";
```

Comment:
Possible NRE here? `tracked == null` is checked above but this is reached anyway.